lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/oauth/OIDCAuthorizationRequest.json

```diff
@@ -10,7 +10,7 @@
 			"description": "space-separated list of scopes"
 		},
 		"claims":{
-			"type":"string",
+			"$ref":"org.sagebionetworks.repo.model.oauth.OIDCClaimsRequest",
```

Comment:
To protect against breaking a client write a test that passes a serialized object to the POJO parser for OIDCClaimsRequest.  If you pass the same serialized object(s) used in ITOIDCTest (name?) then you have maintained the protection already in the code base.